debian/jitsi-meet-prosody.postinst

```diff
@@ -252,6 +245,27 @@ case "$1" in
         if [ "$PROSODY_CONFIG_PRESENT" = "false" ]; then
             invoke-rc.d prosody restart || true
 
+            # give it some time to warm up
+            sleep 10
+
+            if [ "$PROSODY_CREATE_JICOFO_USER" = "true" ]; then
+                # create 'focus@auth.domain' prosody user
+                echo -e "$JICOFO_AUTH_PASSWORD\n$JICOFO_AUTH_PASSWORD" | prosodyctl adduser $JICOFO_AUTH_USER@$JICOFO_AUTH_DOMAIN > /dev/null || true
+
+                # trigger a restart
+                PROSODY_CONFIG_PRESENT="false"
+            fi
+
+            # creates the user if it does not exist
+            echo -e "$JVB_SECRET\n$JVB_SECRET" | prosodyctl adduser jvb@$JICOFO_AUTH_DOMAIN > /dev/null || true
+
+            # Make sure the focus@auth user's roster includes the proxy component (this is idempotent)
+            prosodyctl mod_roster_command subscribe focus.$JVB_HOSTNAME $JICOFO_AUTH_USER@$JICOFO_AUTH_DOMAIN
+
+            # To make sure the roster command is loaded
+            sleep 1
```

Comment:
Is the sleep really necessary?

Reply:
Not sure, but as it was necessary a wait for the adduser which uses shell in the new version ... a small wait before restarting will be on the safe side. At some point in the future we can start using prosodyctl shell for that and get rid of the restart, I will drop a comment and for that.